.buildkite/build.rayci.yml

```diff
@@ -34,3 +34,13 @@ steps:
       - forge
       - raypy38cu118base
     job_env: forge
+
+  - label: ":tapioca: build: pip-compile dependencies"
+    instance_type: small
+    commands:
+      - rm ./python/requirements_compiled.txt
+      - ./ci/ci.sh compile_pip_dependencies
+      - cp -f ./python/requirements_compiled.txt /artifact-mount/var/
```

Comment:
why not just put this into /artifact-mount?

Reply:
This is running with non-root privileges. It looks like we can't properly modify the access flags for the root mountpoint (at least my attempts to do that failed). So a subdirectory was the easiest way to make this work.